proxy/http/HttpSM.cc

```diff
@@ -5609,7 +5610,7 @@ HttpSM::attach_server_session(HttpServerSession *s)
   hsm_release_assert(server_entry == NULL);
   hsm_release_assert(s->state == HSS_ACTIVE);
   server_session = s;
-  server_session->transact_count++;
+  server_transact_count = server_session->transact_count++;
```

Comment:
this is redundant, but we need the additional value because the server_session is gone at logging time